### PR TITLE
Sprint2 task1

### PR DESCRIPTION
**Pull request by DamianWieszczak**
- reorganization of controllers and views,
- creating an admin,
- admin login/logout
- added rspec test for admin login

### CODE REVIEW

File: app/models/admin.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Admin < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (piowit) [inline app/models/admin.rb]: admin shouldn't be able to register

File: app/views/admin/pages/dashboard.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h1>Admins Dashboard</h1>
+
+<div class="card text-white bg-dark mb-3" style="max-width: 18rem;">
+  <div class="card-header">Admins Panel</div>
+  <div class="card-body">
+    <h5 class="card-title">Creating Kudo</h5>
+    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+  </div>
+</div>
```
Comment (piowit) [inline app/views/admin/pages/dashboard.html.erb]: why we have two the same vies for dasboard?

File: app/views/admins/pages/dashboard.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h1>Admins Dashboard</h1>
+
+<div class="card text-white bg-dark mb-3" style="max-width: 18rem;">
+  <div class="card-header">Admins Panel</div>
+  <div class="card-body">
+    <h5 class="card-title">Creating Kudo</h5>
+    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+  </div>
+</div>
```
Comment (piowit) [inline app/views/admins/pages/dashboard.html.erb:9]: why we have two the same vies for dasboard?

File: spec/system/admin_sign_in_spec.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Sign in process', type: :system do
+  before do
+    driven_by(:rack_test)
```
Comment (piowit) [inline spec/system/admin_sign_in_spec.rb]: if you want you can put this line inside rails helper - it will run before every test that require rails_helper

**Review by piowit — COMMENTED** [review]
one unnecessary file, otherwise looks good :)

File: app/views/admin/pages/dashboard.html.erb (hunk 2 above)
Comment (DamianWieszczak) [reply app/views/admin/pages/dashboard.html.erb]: Right, totally unnecessary view on the way back when renaming from admin to admins

File: app/models/admin.rb (hunk 1 above)
Comment (DamianWieszczak) [reply app/models/admin.rb]: This option has been corrected and removed

File: spec/system/admin_sign_in_spec.rb (hunk 4 above)
Comment (DamianWieszczak) [reply spec/system/admin_sign_in_spec.rb]: As suggested, I moved these lines to rails_helper and removed them from the tests

**Review by piowit — APPROVED** [review]

**Pull request merged**